components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -351,6 +352,7 @@
                     <%
                         for (String classApply : classApplies) {
                             if (selectedClassApplied != null && classApply.equals(selectedClassApplied)) {
+                                classApply = Encode.forHtmlAttribute(classApply);
                     %>
                     <option value="<%=classApply%>" selected="selected"><%=classApply%>
```

Comment:
Here 'classApply' is used in two HTML contexts, i.e as HTML attribute and as HTML text content. So if you are only using attribute encoding when it comes to the HTML text content, you are ignoring characters that are significant in HTML content to be encoded and encode some others instead. So use appropriate methods, or encode with Encode.forHtml() which encodes all significant characters.